insights-notification-schemas-java/src/main/java/com/redhat/cloud/notifications/ingress/Parser.java

```diff
@@ -0,0 +1,142 @@
+package com.redhat.cloud.notifications.ingress;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.node.JsonNodeType;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.networknt.schema.ApplyDefaultsStrategy;
+import com.networknt.schema.JsonMetaSchema;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SchemaValidatorsConfig;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidatorTypeCode;
+import com.networknt.schema.ValidationResult;
+import com.redhat.cloud.notifications.validator.LocalDateTimeValidator;
+
+import java.io.UncheckedIOException;
+
+public class Parser {
+
+    private final static ObjectMapper objectMapper = new ObjectMapper();
+    private final static JsonSchema jsonSchema;
+
+    private final static String CONTEXT_FIELD = "context";
+    private final static String EVENTS_FIELD = "events";
+    private final static String PAYLOAD_FIELD = "payload";
+
+    static {
```

Comment:
isn't this something we should use in a separate bean? At least in Spring Boot I would have done that.

Reply:
Could you elaborate? 
I just want to initialize some properties of the `ObjectMapper`  and the `jsonSchema` - I can put this code below in a private static function and call it from here if that's better. I know nothing about Spring Boot.

Reply:
My bad. This is a plain Java SE project.